pkg/controller/initializer/stash/controller.go

```diff
@@ -51,10 +52,11 @@ func NewController(
 }
 
 type restoreInfo struct {
-	invoker core.TypedLocalObjectReference
-	target  *v1beta1.RestoreTarget
-	phase   v1beta1.RestorePhase
-	do      dmcond.DynamicOptions
+	invoker    core.TypedLocalObjectReference
+	target     *v1beta1.RestoreTarget
+	phase      v1beta1.RestorePhase
+	do         dmcond.DynamicOptions
+	invokerUID types.UID
```

Comment:
Is is necessary anymore?

Reply:
Useful information for users/developers I guess.